src/components/Search/SearchResultsList.tsx

```diff
@@ -7,15 +9,43 @@ import SearchResultsListItem from './SearchResultsListItem';
 function SearchResultsList(props: SearchResultsListProps) {
   const { searchResults, view } = props;
 
+  useEffect(() => {
```

Comment:
Is this useEffect really necessary? How would you accomplish the same result without it?

Reply:
I used `useEffect` hook to get the `search-results-list` element once it is mounted and then calculate the amount of height it can occupy. Should I use a different approach?

Reply:
It's not whether _you should_ use a different approach, but thinking through whether it's necessary or not to accomplish your goal.

Reply:
I think the `useEffect` hook is necessary since it will help me get the element once it is mounted, is there a reason why you think I shouldn't use it?